src/Host.cpp

```diff
@@ -1913,73 +1913,77 @@ bool Host::uninstallPackage(const QString& packageName, int module)
     return true;
 }
 
-void Host::readPackageConfig(const QString& luaConfig, QString& packageName)
+void Host::readPackageConfig(const QString& luaConfig, QString& packageName, int isModule)
 {
-    QFile configFile(luaConfig);
-    QStringList strings;
-    if (configFile.open(QIODevice::ReadOnly | QIODevice::Text)) {
-        QTextStream in(&configFile);
-        while (!in.atEnd()) {
-            strings += in.readLine();
-        }
+    QString newName = getPackageConfig(luaConfig, isModule);
+    if (!newName.isEmpty()){
+        packageName = newName;
     }
+}
+
+QString Host::getPackageConfig(const QString& luaConfig, int isModule)
+{
+    QString packageName{QString()};
```

Comment:
This is pointless, the default constructor for a QString gives an empty one, just use:
```suggestion
    QString packageName;
```